terminology/glossary/Process and Management/Vulnerability Disclosure/CVE/Snyk.md

```diff
@@ -0,0 +1,9 @@
+# CVE
+
+## Description
+
+CVE stands for Common Vulnerabilities and Exposures. CVE is a free service that identifies and catalogs known software or firmware vulnerabilities. CVE is not, in itself, an actionable vulnerability database. It is, in effect, a standardized dictionary of publicly known vulnerabilities and exposures. CVE is used by many security-related products and services such as vulnerability management and remediation, intrusion detection, incident management, and more.
+
+## Source
+
+[Snyk](https://snyk.io/learn/what-is-cve-vulnerablity/)
```

Comment:
I've added CVE.org's definition, and kept snyk's since the definition itself is vendor neutral, and as ran-dall said, in the future we possibly will have more member organization contributions to this,

Reply:
This feels strange, but I'll leave my dissent with this... 

To a vendor (Sonatype, my current employer, is active in this org), any link from a source represents a backlink to that vendor. Backlinks directly improve SEO, especially if the backlink originator is a reputable source with good SEO (like OpenSSF). 

To avoid the optics of favorability, when there are reputable, neutral sources available (Wikipedia, NIST, and in this case, the originator of the term), I think we should prioritize those (again, my current company would [benefit from this not being the case](https://ossindex.sonatype.org/doc/cve#:~:text=Common%20Vulnerabilities%20and%20Exposures,for%20publicly%20known%20cybersecurity%20vulnerabilities.)). However, that's not my ask or intent. A glossary in particular should lean towards objectivity.

I definitely understand the potential for open contribution and across other content OpenSSF will create I think there's certainly opportunity. But to the point regarding backlinks, this should be vetted, else we risk trying to play gatekeeper or just leaving the gate open. For example, below is just a handful of other Vendors with similar pages. Would it serve a user to have seven+ definitions?

[RedHat - What is a CVE](https://www.redhat.com/en/topics/security/what-is-cve)
[Balbix - What is a CVE](https://www.balbix.com/insights/what-is-a-cve/)
[Imperva](https://www.imperva.com/learn/application-security/cve-cvss-vulnerability/)
[Upguard - What is a CVE](https://www.upguard.com/blog/cve)

Whatever we choose, I'll support it, just wanted to call this out since we are still early in mapping out content and contribution.

Reply:
@thiswayman Thank you for taking the time to write that out.

> To a vendor (Sonatype, my current employer, is active in this org), any link from a source represents a backlink to that vendor. Backlinks directly improve SEO, especially if the backlink originator is a reputable source with good SEO (like OpenSSF).

I hadn't thought far ahead, but it's a good point with the SEO; this shouldn't be seen as a marketing opportunity. 

> To avoid the optics of favorability, when there are reputable, neutral sources available (Wikipedia, NIST, and in this case, the originator of the term), I think we should prioritize those (again, my current company would [benefit from this not being the case](https://ossindex.sonatype.org/doc/cve#:~:text=Common%20Vulnerabilities%20and%20Exposures,for%20publicly%20known%20cybersecurity%20vulnerabilities.)). 

I agree with the spirit of what you're trying to convey, but I think we need to be careful here. Just because something came from a 'neutral source' doesn't make it more accurate or descriptive. I prefer to leave the ordering to search relevance, or perhaps we can implement a simple upvote system. Now, in the absence of such data, could we have a default order that prefers 'neutral sources, but I'd only use it sparingly, as mentioned. (Hope that makes sense)

---

I think it's also important to note here...
	- From a Graph (database) perspective, the nodes representing a lexicon entry will (for the most part) be the same, regardless of their source. So strictly from a Graph (or database) perspective, the source is trivial at best.

Reply:
I propose we take this point with the team on our next education SIG call.

Reply:
after raising this issue to the education SIG, we have agreed to keep the glossary as vendor-neutral as possible. i've removed the definition by Snyk and replaced it with ones from cve.org, NIST and wikipedia.
resolving